miio/integrations/vacuum/dreame/dreamevacuum_miot.py

```diff
@@ -190,37 +274,132 @@ def volume(self) -> str:
     def voice_package(self) -> str:
         return self.data["voice_package"]
 
+    @property
+    def timezone(self) -> str:
+        return self.data["timezone"]
+
+    @property
+    def cleaning_time(self) -> str:
+        return self.data["cleaning_time"]
+
+    @property
+    def cleaning_area(self) -> str:
+        return self.data["cleaning_area"]
 
-class DreameVacuumMiot(MiotDevice):
-    """Interface for Vacuum 1C STYTJ01ZHM (dreame.vacuum.mc1808)"""
+    @property
+    def first_clean_time(self) -> str:
+        return self.data["first_clean_time"]
 
-    mapping = _MAPPING
-    _supported_models = ["dreame.vacuum.mc1808"]
+    @property
+    def total_clean_time(self) -> str:
+        return self.data["total_clean_time"]
+
+    @property
+    def total_clean_times(self) -> str:
+        return self.data["total_clean_times"]
+
+    @property
+    def total_clean_area(self) -> str:
+        return self.data["total_clean_area"]
+
+    @property
+    def cleaning_mode(self):
+        cleaning_mode = self.data["cleaning_mode"]
+        if self.model == DREAME_1C:
+            try:
+                return CleaningModeDreame1C(cleaning_mode)
+            except ValueError:
+                _LOGGER.error(f"Unknown CleaningMode ({cleaning_mode})")
+                return CleaningModeDreame1C.Unknown
+        elif self.model in [DREAME_D9, DREAME_F9]:
+            try:
+                return CleaningModeDreameF9(cleaning_mode)
+            except ValueError:
+                _LOGGER.error(f"Unknown CleaningMode ({cleaning_mode})")
+                return CleaningModeDreameF9.Unknown
+
+    @property
+    def life_sieve(self) -> Optional[str]:
+        return self.data.get("life_sieve")
+
+    @property
+    def life_brush_side(self) -> Optional[str]:
+        return self.data.get("life_brush_side")
+
+    @property
+    def life_brush_main(self) -> Optional[str]:
+        return self.data.get("life_brush_main")
+
+    # TODO: get/set water flow for Dreame 1C
+    @property
+    def water_flow(self) -> Optional[WaterFlow]:
+        try:
+            water_flow = self.data["water_flow"]
+        except KeyError:
+            return None
+        try:
+            return WaterFlow(water_flow)
+        except ValueError:
+            _LOGGER.error("Unknown WaterFlow (%s)", self.data["water_flow"])
+            return WaterFlow.Unknown
```

Comment:
```suggestion
        try:
            return WaterFlow(self.data["water_flow"])
        except KeyError:  # device does not report water flow
            return None
        except ValueError:
            _LOGGER.error("Unknown waterflow reported: %s", self.data["water_flow"])
            return None
```
I think it's better to log an error and return 'None` is better than having a separate unknown state. From the perspective of a library user, it is easier to check for `None` and the warning will give away that something unexpected has happened.